src/app/api/events/transactions/route.ts

```diff
@@ -0,0 +1,90 @@
+import { TransactionDetailsFormData } from '@/types/meetings';
+import { NextRequest, NextResponse } from 'next/server';
+
+export async function POST(request: Request | NextRequest) {
+  const url = new URL(request.url);
+
+  const bodyData = (await request.json()) as
+    | TransactionDetailsFormData
+    | undefined;
+  const action = url.searchParams.get('action');
+
+  console.log({ bodyData }); // TODO: remove console.log
```

Comment:
nit: let's take out console.log

Reply:
yeah this will be removed when adding real data